lib/router.js

```diff
@@ -96,11 +96,31 @@ const guardAuth = (router) => router.beforeEach((to, from, next) => {
 
 
 
+////////////////////////////////////////////////////////////////////////////////
+// ALERTS
+
+// Hides the alert of the App component. Any modal should be destroyed upon a
+// route change, so we do not need to hide modal alerts, and there are no other
+// alerts. (We may need to modify this if we add other alerts.)
+const hideAppAlert = (router) => router.afterEach(() => {
+  const root = router.app;
+  // When using mockRoute() in testing, `root` is the App component. In
+  // production, `root` is the parent of the App component, though it might not
+  // have children components before the first render.
```

Comment:
is this discrepancy fixable at the root?

Reply:
I think so — it'd entail changing `mockRoute()`. I think that'll definitely be needed if/when we use `router.app` in other places, and may be worth it now to clean up this code. What do you think?

Reply:
I merged the PR, but definitely let me know if you think we should create an issue to try to change this sooner rather than later.